db/dcrpg/pgblockchain.go

```diff
@@ -1397,31 +1397,23 @@ func (pgb *ChainDB) DevBalance() (*dbtypes.AddressBalance, error) {
 // and if cache is stale or missing data for the address, a DB query is used. A
 // successful DB query will freshen the cache.
 func (pgb *ChainDB) AddressBalance(address string) (*dbtypes.AddressBalance, error) {
-	numSpent, numUnspent, amtSpent, amtUnspent, err :=
-		pgb.AddressSpentUnspent(address)
+	balance, err := pgb.AddressSpentUnspent(address)
```

Comment:
Seems like the whole function now reduces to `return pgb.AddressSpentUnspent(address)`